### PR TITLE
multiple embed blocks fix

### PR DESCRIPTION
**Pull request by craescu**
Hi

Here's the pull request for version 1.x.
Thanks!

### CODE REVIEW

**Comment by dbu** [comment]
we get an error in the php 5.3 build: https://travis-ci.org/symfony-cmf/block-bundle/jobs/138818271

likely php 5.3 is not supported by one of the dependencies and an older version installed that is missing a feature. prefer-lowest also fails. so i think we should do a new minor version in this pr and drop php 5.3 support and only have php 5.4. and figure out what minimal version of whatever it is that triggers the error we need.

or we simply do the feature only for version 2 of the bundle.

**Comment by craescu** [comment]
Yes, I'm aware of it. Trying to overcome it. I don't have php 5.3 installed cause I haven't used it in the last 3 years.

**Comment by dbu** [comment]
me neither. but thats why we have travis build that test matrix so we
realize it before people stuck on old projects get unexpected failures.

bumping the php version in composer.json and .travis.yml is a valid
solution for me in this case. people on legacy projects then just stop
getting updates which is acceptable. they don't get php 5.3 updates
anyways...

**Comment by craescu** [comment]
php5.3 pass now but 5.5 and 5.6 fail.

File: Templating/Helper/CmfBlockHelper.php [hunk 1]
```diff
@@ -12,7 +12,7 @@
 namespace Symfony\Cmf\Bundle\BlockBundle\Templating\Helper;
 
 use Symfony\Component\Templating\Helper\Helper;
-use Symfony\Component\HttpKernel\Log\LoggerInterface;
+use Psr\Log\LoggerInterface;
```
Comment (dbu) [inline Templating/Helper/CmfBlockHelper.php]: you can't do this in an 1.x branch as its a BC break.

**Comment by dbu** [comment]
the timeouts are likely fixed by changing .travis.yml env setting to
```
env:
  global: SYMFONY_DEPRECATIONS_HELPER=weak
  matrix: SYMFONY_VERSION=2.8.*
```

**Comment by craescu** [comment]
Is this ok now?

**Comment by dbu** [comment]
great, thanks! added a changelog note and released as 1.3.1

merging 1.3 to master in #255, once thats done we can wrap up #252

**Pull request merged**